modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/display_layout/edit_cp_definition_cp_display_layout.jsp

```diff
@@ -74,7 +74,7 @@ String searchContainerId = "CPDefinitionsSearchContainer";
 		<aui:model-context bean="<%= cpDisplayLayout %>" model="<%= CPDisplayLayout.class %>" />
 
 		<div class="sheet">
-				<div class="panel-group panel-group-flush">
+			<div class="panel-group panel-group-flush">
```

Comment:
Did you really change this or was it by mistake?

Reply:
Not really, probably a change from SF.